Submodels/Parameterizations.py

```
@@ -0,0 +1,775 @@
+        ----------
+        reference_file : str
+            Path to the file containing the reference profile coordinates.
+        plot : bool
+            Boolean controlling if the initial parameterization is plotted. Default is False.
+
+        Returns
+        -------
+        Tuple of optimized Bezier control points for the airfoil parameterization.
+        """
+
+        def Objective(x: list[float]
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)